### DISCUSSION TITLE
Prose initial value 640px < breakpoints

### DISCUSSION THREAD

**Discussion by xaviernouaille** [tailwindlabs/tailwindcss#7673] [Help] [answered]
Hello,
I recently discovered the prose property offered by tailwind. I'm supoer happy because it will allow me to manage the texts of all my pages and keep them consistent overall. Everything works perfectly, however, I have the impression that there are no default rules for fonts.
To illustrate my point, here is my code, and below 640px, the text is the same size (p, h1, h2, h3, h4). I wanted to know if it is up to me to determine default values ​​for the tags. Is there a way to have below 640px, a stylized text.

``` import { ReactElement } from 'react'

const Test = (): ReactElement => {
  return (
    <article className='prose prose-base prose-slate prose-headings:font-semibold sm:prose-base lg:prose-lg xl:prose-xl 2xl:prose-2xl dark:prose-invert'>
      <h1>Garlic bread with cheese: What the science tells us</h1>

      <h2>h2</h2>
      <h3>h3</h3>
      <h4>h4</h4>
      <p>
        For years parents have espoused the health benefits of eating garlic
        bread with cheese to their children, with the food earning such an
        iconic status in our culture that kids will often dress up as warm,
        cheesy loaf for Halloween.
      </p>
      <p>
        But a recent study shows that the celebrated appetizer may be linked to
        a series of rabies cases springing up around the country.
      </p>
    </article>
  )
}

export default Test
```

![image](https://user-images.githubusercontent.com/68466322/155812841-c6983de1-795b-451f-b0ea-19f7c9397c87.png)
![image](https://user-images.githubusercontent.com/68466322/155812888-b279d1a6-f01d-4f20-908c-bdadfc289bbd.png)

Thanks for reading me !

**Comment by xaviernouaille** [comment]
up please :smile:

**Comment by adamwathan** [comment, marked as answer]
Hey! The `prose` class does style things below 640px — your code works the way I'd expect when I put it in Tailwind Play:

https://play.tailwindcss.com/7p7Hmngluc?size=368x720

<img width="802" alt="image" src="https://user-images.githubusercontent.com/4323180/155976260-496873c1-f363-408c-afd0-1d9d7cababe1.png">

If you can provide a full reproduction it will be easier to figure out where things are going wrong in your project.

**Reply by xaviernouaille** [reply]
Thank you !!
